app/dashboard/templates/shared/sidebar_search.html

```diff
@@ -84,6 +84,42 @@
       </div>
     {% endif %}
 
+    {% if profile_id and user.profile.is_moderator or user.profile.is_staff %}
+    <div class="col">
```

Comment:
Could we indent the by 1 ?

Reply:
I guess we can.  I typically don't indent things inside of the `{% if %}` blocks because the indentation carries over into the html.